modules/tools/vehicle_calibration/plot_results.py

```diff
@@ -34,23 +34,23 @@
     sys.exit(0)
 
 with open(sys.argv[1], 'r') as f:
-   cmd_table = {}
+    cmd_table = {}
 
     for line in f:
-        items = line.split(',')
-        cmd = round(float(items[0]))
-        speed = float(items[1])
-        acc = float(items[2])
-        if cmd in cmd_table:
-            speed_table = cmd_table[cmd]
-            if speed in speed_table:
-                speed_table[speed].append(acc)
+            items = line.split(',')
```

Comment:
This indent is wrong.

Reply:
It's resolved. autopep8 may have gotten confused, but I think python would run it the same: https://github.com/hhatto/autopep8/issues/486.